docs/demos/mpi.ipynb

```diff
@@ -0,0 +1,307 @@
+{
```

Comment:
<p>You should be able to delete this, if you add kwargs to the first function you defined.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/steven-murray/powerbox/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>